docs/pages/api/binaries/latest.ts

```diff
@@ -0,0 +1,73 @@
+import type { NextRequest } from "next/server";
+
+const REGISTRY = "https://registry.npmjs.org";
+const SUPPORTED_PACKAGES = new Set(["turbo"]);
+const TAG = "latest";
+
+const ERRORS = {
+  NOT_FOUND: {
+    status: 404,
+    error: "Not Found",
+  },
+  INTERNAL_SERVER_ERROR: {
+    status: 500,
+    error: "Internal Server Error",
+  },
+};
+
+export default async function handler(req: NextRequest) {
+  const { searchParams } = new URL(req.url);
```

Comment:
This is only ever intended to be invoked programmatically; it doesn't necessarily need to be human-readable.

I'm tempted to make it POST a body of JSON to make it easier to tag on things.

Reply:
I hear you, if we want to do experiment style rollouts in the future it would be nicer to be able to just ship a JSON blob of data that we can match on. But using GET here is semantically correct and query params will take us a long way in my opinion.

Reply:
Going with POST